components/src/input/tex/extensions/all-packages/all-packages.js

```diff
@@ -0,0 +1,38 @@
+import './lib/all-packages.js';
+
+import {AllPackages} from '../../../../../../mathjax3/input/tex/AllPackages.js';
+import {insert} from '../../../../../../mathjax3/util/Options.js';
+
+Loader.preLoad(
+    '[tex]/action',
+    '[tex]/ams',
+    '[tex]/ams_cd',
```

Comment:
Yes, as per our discussion of this a week or two ago.  Because `autobold` uses direct access to the DOM (rather than access through a `DOMadaptor`), and because it causes a page refresh for each equation processed, I think it should be deprecated.  It was probably not a great idea in the first place, and this seems like a good time to drop it.